slack-birthdays.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+date_default_timezone_set('America/New_York');
+
+function getBirthdays($connection, $date)
```

Comment:
why make this a function if it's only used in one place and is only three lines long?

Reply:
Force of habit. I'll fix that. :)